MinPlatformPkg/Test/Library/TestPointCheckLib/StandaloneMmCheckCommunicationBuffer.c

```diff
@@ -0,0 +1,120 @@
+/** @file
+
```

Comment:
This file seems to be functionally identical to the SMM one? Let's consolidate them

Reply:
Consolidated.